File: 9-Palindrome Number-回文数/9-Palindrome Number-回文数.js
/**
 * @param {number} x
 * @return {boolean}
 */
var isPalindrome = function(x) {
    let s = String(x)
    let len = s.length
    for(i = 0; i < Math.floor(len / 2); i++){
        if(s[i] !== s[len - i - 1]){
            return false
        }
    }
    return true
};

File: 9-Palindrome Number-回文数/9-Palindrome Number-回文数.java
class Solution {
    public boolean isPalindrome(int x) {
        String s = String.valueOf(x);
        int len = s.length();
        for(int i = 0; i < Math.floor(len / 2); i++){
            if(s.charAt(i) != s.charAt(len - i - 1)){
                return false;
            }
        }
        return true;
    }
}
